compiler/rustc_session/src/options/enforceable_mitigations.rs

```diff
@@ -0,0 +1,182 @@
+use std::collections::BTreeSet;
```

Comment:
I propose changing this module to mitigation_coverage instead.

Reply:
done